### ISSUE TITLE
Re-word "search limited" message and fix broken links

### ISSUE THREAD

**Issue by ryan-blunden** [sourcegraph/sourcegraph#4256]
![search-limited](https://user-images.githubusercontent.com/133014/58589940-e0452f80-8217-11e9-800c-9eb2aebb0bf6.gif)

**TL;DR**

- Re-word the "search limited performance and accuracy" message
- Confirm why 100 was chosen, confirm why and when accuracy is affected
- Fix the two broken links in the message
- Reconsider only showing to admins and, not to all users

---

When more than 100 repositories are synced to a single node instance, this message appears:

> Search performance and accuracy are limited on single-node Docker deployments. We recommend that instances with 100+ repositories deploy to a cluster for optimal performance. Contact us for support or to learn more.

This gives the impression that search doesn't work properly if more than 100 repositories synced. That doesn't seem right, considering the available size of instances on any cloud provider.

Also, this is shown to all users. I think this should be limited to admins.

In my gif, the query is `file:package.json type:diff repo:sourcegraph/sourcegraph$` which I think Sourcegraph can handle just fine. If it was a query that timed out or used selectors that were performance intensive, then I could understand why a message might be shown.

I think this needs to be re-written to something like "Search performance **may be impacted** on single-node Docker deployments with 100+ repositories." as it's not an absolute.

Regarding accuracy. how did we come up with the number 100 and is there a way for the system to self-diagnose when accuracy is affected, and then, to show a message specifically for that?

**Comment by slimsag** [comment]
Context / discussions around the original change:

- https://github.com/sourcegraph/sourcegraph/issues/2176#issuecomment-484762332
- https://github.com/sourcegraph/sourcegraph/pull/3374
- https://github.com/sourcegraph/sourcegraph/issues/3370

**Comment by malomarrec** [comment]
Tagging similar issue in #20138 to feed into prioritisation work.

**Comment by malomarrec** [comment]
This sounds like a starving issue. Should I close it?

**Comment by fkling** [comment]
Closing this as no longer relevant because the UI (and infra) has changed a lot since then.
